lib/timetable-form/Form.tsx

```diff
@@ -1,10 +1,61 @@
+import { useState } from "react";
 import styles from "./styles.module.css";
 
+const stageNames = [
+  "Timetable published",
+  "Draft plan for public consultation published",
+  "Public consultation start",
+  "Public consultation end",
+  "Submit plan for examination",
+  "Examination hearing start",
+  "Planning inspectorate examination start",
+  "Planning inspectorate examination end",
+  "Planning inspectorate found sound",
+  "Examination hearing end",
+  "Inspector report published",
+  "Plan adopted",
+] as const;
+
+type StageName = (typeof stageNames)[number];
+
+const defaultDate = "";
+
+type Stages = {
+  [key in StageName]: string;
+};
+
+const initialStages = Object.fromEntries(
+  stageNames.map((stage) => [stage, defaultDate])
+) as Stages;
+
 export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
   const { className, ...otherProps } = props;
+  const [lpa, setLpa] = useState<string>("");
+  const [stages, setStages] = useState<Stages>(initialStages);
+
   return (
     <div className={`${className} ${styles.form}`} {...otherProps}>
-      <h1>Form Title </h1>
+      <h1 data-testid="form-title">Timetable Form </h1>
+      <div className={styles.formContent}>
+        Local Planning Authority
+        <input value={lpa} onChange={(e) => setLpa(e.target.value)} />
+        {Object.entries(stages).map(([stageName, date]) => (
+          <>
+            {stageName}
+            <input
+              key={stageName}
```

Comment:
This `key` should be on the outer component, which means you'll have to use the longhand for a fragment (e.g. `<Fragment key={stageName}>...</Fragment>`

Reply:
you are right! I had added the prop to get rid of the "Each child in a list should have a unique "key" prop" but it's definitely on the wrong element. I will change